internal/pkg/template/templates/app/cf.yml

```diff
@@ -148,3 +148,6 @@ Outputs:
   TemplateVersion:
     Description: Required output to force the stackset to update if mutating version.
     Value: {{.TemplateVersion}}
+  StackSetOpId:
+    Description: Required output to forces the stackset to update on every operation
```

Comment:
This explains why we need the output (to force stackset updates), but why do we need to force the stackset update in the first place? Should we add that as a comment?

Reply:
good idea - reasoning is in the PR description (3) but i'll document it in the template as well.